generated/gstreamer/gst/mpegts/AtscMultString.d

```diff
@@ -27,6 +27,7 @@ module gst.mpegts.AtscMultString;
 private import glib.MemorySlice;
 private import glib.PtrArray;
 private import glib.Str;
+private import glib.c.functions;
```

Comment:
Do you know of a reason why a lot of files have this import added? Or is it just an artifact from using a newer version of gir-to-d?

Reply:
I didn't add it myself. If you want to know my girtod's version:
`girtod --version
GIR to D 0.19.1
`

Reply:
The latest version is 0.22.0 so that could explain the difference.

Reply:
I built girtod from Github: v0.22.0-3-g4a5265f
But nothing was changed...